delivery.yaml

```diff
@@ -0,0 +1,42 @@
+build_steps:
+    - desc: Prepare Environment
+      cmd: |
+        apt-get update
+        apt-get install -y python3 python3-pip jq
+        pip3 install scm-source
+
+    - desc: Install Docker
+      cmd: 'curl -sSL https://get.docker.com/ | sh'
+
+    - desc: scm-source file create
+      cmd: |
+        scm-source
```

Comment:
Per the [CDP docs](https://cloud.docs.zalando.net/concepts/builds/#scm-source), this should not be necessary:

> CDP will automatically inform Pierone of the SCM Source information on push. This means the /scm-source.json file does not need to be included in the pushed image.

Reply:
This is for compatibility, in case if I want to build and push it locally.